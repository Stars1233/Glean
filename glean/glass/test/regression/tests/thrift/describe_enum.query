action: describeSymbol
args: "test/thrift/lib.thrift/SymbolKind"
