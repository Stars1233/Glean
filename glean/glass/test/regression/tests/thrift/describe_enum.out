[
    "@generated",
    {
        "comments": [],
        "contains_relation": {
            "firstChild": "nondeterministic",
            "firstParent": "nondeterministic",
            "hasMoreChildren": false,
            "hasMoreParents": false
        },
        "extends_relation": {
            "firstChild": "nondeterministic",
            "firstParent": "nondeterministic",
            "hasMoreChildren": false,
            "hasMoreParents": false
        },
        "kind": 12,
        "language": 9,
        "location": {
            "filepath": "lib.thrift",
            "range": {
                "columnBegin": 1,
                "columnEnd": 2,
                "lineBegin": 163,
                "lineEnd": 194
            },
            "repository": "test"
        },
        "modifiers": [],
        "name": {
            "container": "lib",
            "localName": "SymbolKind"
        },
        "pretty_comments": [],
        "repo_hash": "testhash",
        "signature": "enum lib::SymbolKind",
        "sym": "test/thrift/lib.thrift/SymbolKind",
        "sym_location": {
            "filepath": "lib.thrift",
            "range": {
                "columnBegin": 1,
                "columnEnd": 2,
                "lineBegin": 163,
                "lineEnd": 194
            },
            "repository": "test"
        },
        "sym_other_locations": [],
        "type_xrefs": []
    }
]